model/src/main/java/org/projectnessie/model/IcebergTable.java

```diff
@@ -77,23 +77,12 @@ public Type getType() {
   }
 
   public static IcebergTable of(
```

Comment:
Is this a public API that can be used outside of the project? 
If yes, shouldn't we deprecate that instead of removing it, for backward compatibility purpose?